cmd/mimir/config-descriptor.json

```diff
@@ -168,6 +168,26 @@
           "fieldType": "int",
           "fieldCategory": "advanced"
         },
+        {
+          "kind": "field",
+          "name": "tls_cipher_suites",
+          "required": false,
+          "desc": "Comma-separated list of cipher suites to use. If blank, the default Go cipher suites will be used.",
+          "fieldValue": null,
+          "fieldDefaultValue": "",
+          "fieldFlag": "server.tls-cipher-suites",
+          "fieldType": "string"
+        },
+        {
+          "kind": "field",
+          "name": "tls_min_version",
+          "required": false,
+          "desc": "Minimum TLS version to use. If blank, the Go TLS minimum version will be used.",
```

Comment:
```suggestion
          "desc": "Minimum TLS version to use. If blank, the Go TLS minimum version is used.",
```

Reply:
Note these files are all generated from the upstream text; I included your suggestion and re-generated them.